project_timesheet_time_control/README.rst

```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+==============================
+Project Timesheet Time Control
+==============================
+
+This module adds a button at account analytic line level to compute minutes
```

Comment:
These comments goes for the PR https://github.com/OCA/project/pull/163. I have fixed in that commit (and they will be propagated to this PR). Please read the PR main text, where I explained that there's only one relevant commit for this module.